ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -349,6 +390,33 @@ private[apiserver] final class ApiUserManagementService(
         Future.successful(t)
     }
 
+  private def verifyPartyExistInIdp(
+      rights: Set[UserRight],
+      identityProviderId: IdentityProviderId,
+      isParticipantAdmin: Boolean,
+  ): Future[Unit] =
+    if (isParticipantAdmin) Future.unit
+    else partyExistsOrError(userParties(rights), identityProviderId)
+
+  private def partyExistsOrError(
+      parties: Set[Ref.Party],
+      identityProviderId: IdentityProviderId,
+  ): Future[Unit] =
+    partyRecordExist(identityProviderId, parties)
```

Comment:
```suggestion
    partyRecordsExist(identityProviderId, parties)
```
or (imho) use a method named `partyRecordsExist` instead of `apply` and then we'll get consistent names for this operation for free